Assets/Scripts/UI/Canvas/PhotoTaking/PhotoTaker.cs

```diff
@@ -34,6 +35,12 @@ public sealed class PhotoTaker : CameraPermission
         
         private void Awake() => FindCamera();
 
+        private void Update()
+        {
+            if(_webcamTexture && _webcamTexture.isPlaying)
+                Graphics.Blit(_webcamTexture, renderTexture);
+        }
+
```

Comment:
Dit zou beter in de [Camera panel class](https://github.com/Team-Swamp/Codename-BDLMTW/blob/develop/Assets/Scripts/UI/Canvas/PhotoTaking/CameraPanel.cs) kunnen, want daar wordt het gebruikt op line 25.

Ook zou dit beter aan en uit gezet kunnen worden via `OnEnable` & `OnDisable`. Voor de rest ziet het er prima uit.